tests/functional/tests/eviction/test_eviction.py

```diff
@@ -21,8 +21,7 @@
 
 @pytest.mark.parametrize("cls", CacheLineSize)
 @pytest.mark.parametrize("mode", [CacheMode.WT, CacheMode.WB, CacheMode.WO])
-@pytest.mark.xfail  # TODO: remove when fixed
-def test_write_size_greater_than_cache(pyocf_ctx, mode: CacheMode, cls: CacheLineSize):
+def test_orig(pyocf_ctx, mode: CacheMode, cls: CacheLineSize):
     """Test if eviction does not occur when IO greater than cache size is submitted.
     """
     cache_device = Volume(Size.from_MiB(20))  # this gives about 1.375 MiB actual caching space
```

Comment:
I suggest to modify the test to collect cache capacity from the stats rather than assuming the value of 1.375